### PR TITLE
show timezone in dashboard header

### PR DESCRIPTION
**Pull request by trim21**
~~Currently it use `%z` as timezone, not sure how this works with Daylight Time.~~

change it to `%Z` for tz name

image

### CODE REVIEW

File: pgqueuer/cli.py [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
 import os
-from datetime import timedelta
+from datetime import datetime, timedelta
 from typing import Literal
 
 from tabulate import tabulate, tabulate_formats
 
 from . import db, listeners, models, queries, supervisor
 
+__local_tz = datetime.now().astimezone().strftime("%z")
```
Comment (janbjorge) [inline pgqueuer/cli.py]: Could you move `__local_tz` into the `display_stats(...)`? I want to avoid globals when i can.

**Comment by trim21** [comment]
I'm not sure should we use `%z` or `%Z` here, `%Z` may be non-ascii string

**Comment by janbjorge** [comment]
Would `.encode('ascii', 'ignore').decode('ascii')` help?

**Comment by trim21** [comment]
> Would `.encode('ascii', 'ignore').decode('ascii')` help?

no, for example I live in china and on windows it's `中国标准时间`

**Comment by janbjorge** [comment]
How do you feel about an cli-arg ? I like Z more, but z feels safer. Unsure tbh.

```python
python3 -m pgqueuer dashboard --tz-format '%z OR %Z'
```

**Comment by trim21** [comment]
Yes, args like fine, and it should be default `%Z`

`%z` doesn't work with tz in header, if someone live in Daylight Time.

For example `America/Los_Angeles`. it's `2024-12-01 05:00:00-08:00` and `2024-10-29 00:00:00-07:00`

**Comment by trim21** [comment]
maybe default timezone in header is not a good idea

**Comment by janbjorge** [comment]
> maybe default timezone in header is not a good idea

Its getting a bit messy, lets park this it for now.

**Comment by trim21** [comment]
yes, current behaviour looks fine

**Comment by janbjorge** [comment]
> yes, current behaviour looks fine

Great, thanks for submitting.

**Pull request closed without merging**